### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by lmilbaum**
An initial commit for generating a helper container image to be used by the various projects in this organization

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM registry.access.redhat.com/ubi9:9.1.0-1750
+
+RUN dnf install -y dnf-plugins-core-4.1.0-3.el9.noarch make-1:4.3-7.el9.x86_64 && \
```
Comment (RajeshKumarSrivastava) [inline Dockerfile:3]: are we going to install other requirements in the subsequent PRs?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.PHONY: build
+
+ifdef CI
+ENGINE := docker
```
Comment (RajeshKumarSrivastava) [inline Makefile:4]: Just want to know the reason to choose the docker engine for CI here

**Review by RajeshKumarSrivastava — APPROVED** [review]
LGTM

File: Makefile (hunk 2 above)
Comment (lmilbaum) [reply Makefile:4]: The workflow is running in an ubuntu based environment.  docker is the default container engine in Ubuntu.

File: Dockerfile (hunk 1 above)
Comment (lmilbaum) [reply Dockerfile:3]: yup.  You can submit a PR if you feel comfortable with the code base.

**Pull request merged**